### DISCUSSION TITLE
why tokio named tokio?

### DISCUSSION THREAD

**Discussion by vCassius** [tokio-rs/tokio#6382] [General] [answered]
want to know...

**Comment by Darksonn** [comment, marked as answer]
This is explained on [the wikipedia entry](https://en.wikipedia.org/wiki/Tokio_(software)).

> The name "Tokio" is derived from [Tokyo](https://en.wikipedia.org/wiki/Tokyo) and mio.

**Reply by vCassius** [reply]
thx
